### PR TITLE
improvement(aws_config.yaml): change loader type, disk and swap size

### PR DESCRIPTION
**Pull request by yarongilor**
1) in order to avoid java memory issues that failed stresses, the instance type is increased
	   to c5.4xl and swap is changed to 10GB.

	2) Added write stress commands to prepare, to make sure that later read command will not fail
	   for reading a yet-non-written data.

        3) Added parameters of aws_root_disk_size_loader and aws_root_disk_name_loader
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -429,6 +429,12 @@ class SCTConfiguration(dict):
         dict(name="aws_root_disk_name_monitor", env="SCT_AWS_ROOT_DISK_NAME_MONITOR", type=str,
              help=""),
 
+        dict(name="aws_root_disk_size_loader", env="SCT_AWS_ROOT_DISK_SIZE_LOADER", type=int,
+             help=""),
```
Comment (fgelcer) [inline sdcm/sct_config.py]: please, add a precise description of this new attribute inside help paramater

File: sdcm/sct_config.py [hunk 2]
```diff
@@ -429,6 +429,12 @@ class SCTConfiguration(dict):
...
+
+        dict(name="aws_root_disk_name_loader", env="SCT_AWS_ROOT_DISK_NAME_LOADER", type=str,
+             help=""),
```
Comment (fgelcer) [inline sdcm/sct_config.py]: same as above

File: sdcm/sct_config.py [hunk 3]
```diff
@@ -838,7 +844,7 @@ class SCTConfiguration(dict):
     backend_required_params = {
         'aws':  ['user_prefix', "instance_type_loader", "instance_type_monitor", "instance_type_db",
                  "region_name", "security_group_ids", "subnet_id", "ami_id_db_scylla", "ami_id_loader",
-                 "ami_id_monitor", "aws_root_disk_size_monitor", "aws_root_disk_name_monitor", "ami_db_scylla_user",
+                 "ami_id_monitor", "aws_root_disk_size_monitor", "aws_root_disk_name_monitor",  "aws_root_disk_size_loader", "ami_db_scylla_user",
```
Comment (fgelcer) [inline sdcm/sct_config.py]: why is it mandatory?

Comment (yarongilor) [reply sdcm/sct_config.py]: fixed

File: sdcm/sct_config.py (hunk 1 above)
Comment (yarongilor) [reply sdcm/sct_config.py]: what's not self explanatory in the parameter name here? all the similar params have no help text.

**Comment by roydahan** [comment]
@fruch / @bentsi please backport

**Comment by bentsi** [comment]
backported

**Pull request merged**